lib/src/livekit.dart

```diff
@@ -9,6 +9,8 @@ class LiveKitClient {
   /// Convenience method for connecting to a LiveKit server.
   /// Returns a [Room] upon a successful connect or throws when it fails.
   /// Alternatively, it is possible to instantiate [Room] and call [Room.connect] directly.
+  @Deprecated(
+      'Use `Room.connect` instead, This method is deprecated above Protocol v8.')
   static Future<Room> connect(
     String url,
     String token, {
```

Comment:
Can we to default to protocol 7 when using the deprecated connect?

Reply:
ah, let me check this.

Reply:
ok, solved